src/main/java/net/snowflake/ingest/streaming/internal/ChannelRegisterStatus.java

```diff
@@ -24,6 +25,15 @@ Long getStatusCode() {
     return this.statusCode;
   }
 
+  @JsonProperty("message")
+  public String getMessage() {
+    return message == null ? StreamingIngestResponseCode.getMessageByCode(statusCode) : message;
```

Comment:
Should we return some default message instead of null since this will be shown to the customer?

Reply:
`StreamingIngestResponseCode.getMessageByCode` returns 'unknown'.